src/RoutePaths.jsx

```diff
@@ -1,15 +1,16 @@
-import { Error, Explore, Home, Login, Post, Profile, Signup } from "pages";
+import { Bookmarks, Error, Explore, Home, Login, Post, Profile, Signup } from "pages";
 import { Route, Routes } from "react-router-dom";
 
 export function RoutePaths(){
     return (
       <>
         <Routes>
-          <Route path="/" element={<Home />}></Route>
-          <Route path="/explore" element={<Explore />}></Route>
+          {/* <Route path="/" element={<Home />}></Route> */}
+          <Route path="/" element={<Explore />}></Route>
           <Route path="/profile/:userId" element={<Profile />}></Route>
           <Route path="/post/:postId" element={<Post />}></Route>
-          <Route path="/login" element={<Login />}></Route>
+          <Route path="/bookmarks" element={<Bookmarks />}></Route>
+          <Route path="/" element={<Login />}></Route>
```

Comment:
Looks Like you have given the same path for different element, cross-check it once

Reply:
Oh yes, will change this in upcoming branches. Since I was working on explore page, "/" is currently pointing to explore page.